lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -107,7 +107,7 @@ def let_it_be(identifier, **options, &block)
         define_method(identifier) do
           # Trying to detect the context
           # Based on https://github.com/rspec/rspec-rails/commit/7cb796db064f58da7790a92e73ab906ef50b1f34
-          if @__inspect_output.include?("before(:context)") || @__inspect_output.include?("before_all")
+          if /\(:context\)/.match?(@__inspect_output) || @__inspect_output.include?("before_all")
```

Comment:
Maybe, we should be more strict here, e.g., `/(before|after)\(:context\)/`?

Reply:
Ya, I can dig that.. squashed in + pushed up.